### PR TITLE
Fix problem with multiline exception messages

### PR DESCRIPTION
**Pull request by FrankGiesecke**
Only use first line from exception message to build reason phrase.
Multi line message may occur in error as the header does not support new lines.

### CODE REVIEW

File: src/Strategy/JsonStrategy.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ public function getExceptionDecorator(Exception $exception)
             ]));
 
             $response = $response->withAddedHeader('content-type', 'application/json');
-            return $response->withStatus(500, $exception->getMessage());
+            return $response->withStatus(500, strtok($exception->getMessage(), "\n"));
```
Comment (hannesvdvreken) [inline src/Strategy/JsonStrategy.php]: I never heard about this method `strtok`. So I learned something new today. Thank you for that.

**Comment by hannesvdvreken** [comment]
👍  Looks good, thanks for adding tests.

**Comment by philipobenito** [comment]
Just got back from holiday, will get this merged and tagged this week 👍

**Comment by FrankGiesecke** [comment]
@philipobenito Please, remember this PR 😉

**Comment by philipobenito** [comment]
👍

**Pull request merged**